infrastructure/simulation-tool/src/operations/transfer.ts

```diff
@@ -0,0 +1,101 @@
+import { BigNumber, Wallet as EthersWallet, constants } from 'ethers';
+import { Wallet as RollupWallet, Transaction, utils } from '@rsksmart/rif-rollup-js-sdk/';
+import config from '../utils/config.utils';
+import { getRandomBigNumber } from '../utils/number.utils';
+import { TransactionReceipt } from '@rsksmart/rif-rollup-js-sdk/build/types';
+import { sleep } from '@rsksmart/rif-rollup-js-sdk/build/utils';
+import { chooseRandomWallet } from '../utils/wallet.utils';
+
+type PreparedTransfer = Omit<Parameters<RollupWallet['syncTransfer']>[number], 'amount'> & {
+    amount: BigNumber;
+    from: RollupWallet;
+};
+
+type TransferResult = {
+    opL2Receipt: TransactionReceipt;
+    verifierReceipt: TransactionReceipt;
+};
+
+const prepareTransfer = (l2sender: RollupWallet, recipient: EthersWallet): PreparedTransfer => {
+    const [minAmount, maxAmount] = config.weiLimits.transferToNew;
+    return {
+        from: l2sender,
+        to: recipient.address,
+        amount: utils.closestPackableTransactionAmount(getRandomBigNumber(minAmount, maxAmount)),
+        token: constants.AddressZero,
+        nonce: 'committed'
+    };
+};
+
+const executeTransfer = (preparedTransfer: PreparedTransfer): Promise<Transaction> => {
+    const { from, ...transferParameters } = preparedTransfer;
+    // console.log(`Transferring ${transferParameters.amount} RBTC to ${transferParameters.to} ...` );
+
+    return from.syncTransfer(transferParameters);
+};
+
+const geteTransferResult = async (transfer: Transaction): Promise<TransferResult> => {
+    const opL2Receipt = await transfer.awaitReceipt();
+    const verifierReceipt = await transfer.awaitVerifyReceipt();
+
+    return {
+        opL2Receipt,
+        verifierReceipt
+    };
+};
+
+const generateTransfers = (
+    numberOfTransfers: number,
+    funderL2Wallet: RollupWallet,
+    recepients: EthersWallet[]
+): PreparedTransfer[] => {
+    return [...Array(numberOfTransfers)].map(() => prepareTransfer(funderL2Wallet, chooseRandomWallet(recepients)));
+};
+
+const executeTransfers = async (
+    preparedTransfers: PreparedTransfer[],
+    delay: number
+): Promise<Promise<Transaction>[]> => {
+    const executedTx: Promise<Transaction>[] = [];
+    console.log('Executing transfers: ');
+    for (const [i, transfer] of preparedTransfers.entries()) {
+        const promiseOfTransfer = executeTransfer(transfer); // We could make an adapter wallet to use a single execute function for all operations
```

Comment:
For the next PR: not sure if we need an adapter wallet, but I agree that having a single execute function would be better